src/components/selectionBoxLayer.ts

```diff
@@ -0,0 +1,82 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Component {
+  export class SelectionBoxLayer extends AbstractComponent {
+    private _box: D3.Selection;
+    private _boxArea: D3.Selection;
+    private _dismissed = true;
+    private _boxBounds: Bounds = {
+      topLeft: { x: 0, y: 0 },
+      bottomRight: { x: 0, y: 0 }
+    };
+
+    constructor() {
+      super();
+      this.classed("selection-box-layer", true);
+    }
+
+    protected _setup() {
+      super._setup();
+
+      this._box = this._content.append("g").classed("selection-box", true).remove();
+      this._boxArea = this._box.append("rect").classed("selection-area", true);
+    }
+
+    /**
+     * Gets the bounds of the box.
+     *
+     * @return {Bounds} The current bounds of the box.
+     */
+    public bounds(): Bounds;
+    /**
+     * Sets the bounds of the box, and draws the box.
+     *
+     * @param {Bounds} newBounds The desired bounds of the box.
+     * @return {SelectionBoxLayer} The calling SelectionBoxLayer.
+     */
+    public bounds(newBounds: Bounds): SelectionBoxLayer;
+    public bounds(newBounds?: Bounds): any {
```

Comment:
bounds seems to be doing a bit too much to me.  It for one sets the bounds of the box, and it also makes the box appear.

Can we have bounds just set the bounds value and have a separate call to make _dismissed true?